COMPILING.md

```diff
@@ -1,30 +1,43 @@
-# COMPILING #
+COMPILING
+=========
 
-To compile Cuberite from source, you need CMake and make, as well as a C compiler, C++ compiler and linker. To contribute code, you also need a Git client.
+To compile Cuberite from source, you need the following set of software:
 
-## Windows ##
+ * CMake
+ * Platform-specific make tool (windows would be MSVC, linux/osx GNU make, etc.)
+ * C compiler
+ * modern C++ compiler and linker
+
+ To contribute code, you also need a Git client.
+
+Windows
+-------
+
+We use Microsoft Visual Studio for Windows compilation. It is possible to use other toolchains, but we don't test against them and they aren't supported. Visual Studio 2013 Express for Desktop is being actively used for development.
```

Comment:
I think we all have VS2013 Community Edition instead; but that shouldn't be too much of a problem.